src/main/java/org/folio/list/services/CustomTenantService.java

```diff
@@ -50,14 +50,13 @@ protected void afterTenantUpdate(TenantAttributes tenantAttributes) {
       .withTimeout(Duration.of(2, ChronoUnit.MINUTES))
       .build()
       .execute(ctx -> {
-        log.info("Verifying lists are up to date. Attempt #" + (ctx.getRetryCount() + 1));
-        migrationService.verifyListsAreUpToDate();
+        log.info("Performing tenant install migrations. Attempt #" + (ctx.getRetryCount() + 1));
+        migrationService.performTenantInstallMigrations();
         return null;
       }, ctx -> {
-        log.error("Unable to verify lists are up to date", ctx.getLastThrowable());
-        // This RetryTemplate only deals with InsufficientEntityTypePermissionsException, so we know that's what this
```

Comment:
We can't, though — `getLastThrowable` can be _any_ exception thrown from `performTenantInstallMigrations`

Reply:
Ugh, you're right! I must have done something stupid when I tested that >_<